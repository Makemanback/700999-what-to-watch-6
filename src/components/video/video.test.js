import React from 'react';
import {render, screen} from '@testing-library/react';
import Video from './video';

jest.mock(`./video.jsx`, () => function mockVideo() {
  return <div>video</div>;
});

test(`Should Video render correctly`, () => {
  render(<Video />);
  expect(screen.getByText(`video`)).toBeInTheDocument();
});
