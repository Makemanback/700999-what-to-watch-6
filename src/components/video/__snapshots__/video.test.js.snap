// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Video render correctly 1`] = `
<div>
  <video
    alt="video-title"
    autoplay=""
    height="175"
    src="http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4"
    width="280"
  />
</div>
`;
